lib/resolve-decl.js

```diff
@@ -9,16 +12,91 @@ var shallowCloneNode = require('./shallow-clone-node');
 var findNodeAncestorWithSelector = require('./find-node-ancestor-with-selector');
 var cloneSpliceParentOntoNodeWhen = require('./clone-splice-parent-onto-node-when');
 
+var parser = require('postcss-selector-parser');
```

Comment:
Fantastic! I started trying to use this in https://github.com/MadLittleMods/postcss-selector-scope-utility but never got it over the line